src/components/Login.tsx

```diff
@@ -49,6 +52,13 @@ const Login = () => {
     return true; // Return true if the form is valid
   };
 
+  const handleTestLogin = () => {
+    setEmail(testEmail);
+    setAdminLogin(true);
+    setPassword(testPassword);
+    handleSubmit();
+  };
+
```

Comment:
can we make a function which does only login ? and that can be used in both the submit and test account